src/Text/Pandoc/Readers/Odt/StyleReader.hs

```diff
@@ -578,25 +580,30 @@ readListLevelStyles namespace elementName levelType =
 readListLevelStyle :: ListLevelType -> StyleReader _x (Int, ListLevelStyle)
 readListLevelStyle levelType =      readAttr NsText "level"
                                >>?! keepingTheValue
-                                    ( liftA4 toListLevelStyle
-                                      ( returnV  levelType             )
-                                      ( findAttr' NsStyle "num-prefix" )
-                                      ( findAttr' NsStyle "num-suffix" )
-                                      ( getAttr   NsStyle "num-format" )
+                                    ( liftA5 toListLevelStyle
+                                      ( returnV  levelType              )
+                                      ( findAttr' NsStyle "num-prefix"  )
+                                      ( findAttr' NsStyle "num-suffix"  )
+                                      ( getAttr   NsStyle "num-format"  )
+                                      ( findAttr' NsText  "start-value" )
                                     )
   where
-  toListLevelStyle _ p s LinfNone         = ListLevelStyle LltBullet p s LinfNone
-  toListLevelStyle _ p s f@(LinfString _) = ListLevelStyle LltBullet p s f
-  toListLevelStyle t p s f                = ListLevelStyle t      p s f
+  toListLevelStyle _ p s LinfNone b         = ListLevelStyle LltBullet p s LinfNone (startValue b)
+  toListLevelStyle _ p s f@(LinfString _) b = ListLevelStyle LltBullet p s f (startValue b)
+  toListLevelStyle t p s f b                = ListLevelStyle t      p s f (startValue b)
+  startValue (Just v)  = if all isDigit v
+                           then read v
```

Comment:
Is this potentially unsafe?  Suppose v is the empty string, won't read fail?  (Maybe it's guaranteed that this won't happen in an ODT document, but it's still good practice to handle this case.)